### ISSUE TITLE
Missing navigation between Minutes and Logs

### ISSUE THREAD

**Issue by kparal** [fedora-infra/mote#684]
When I display a [minutes summary](https://meetbot.fedoraproject.org/meeting_matrix_fedoraproject-org/2024-01-08/fedora-qa.2024-01-08-16.01.html) of a meeting, there's no option to display the full [meeting logs](https://meetbot.fedoraproject.org/meeting_matrix_fedoraproject-org/2024-01-08/fedora-qa.2024-01-08-16.01.log.html). And vice versa, you can't display the minutes summary when viewing the full logs.

Please hyperlink these two pages, otherwise people have to remember magical keywords to adjust in the URL, or they always need to send/link both URLs, which is not fun. Thank you.

**Comment by Freedisch** [comment]
seems like a quick fix, I will take a look at it

**Comment by raeeceip** [comment]
Hi there! Can i be assigned to this task as well, I would love to help figure out a fix for this

**Comment by Precious000** [comment]
Hi  I’ll like to explore this task.

**Comment by gridhead** [comment]
@Freedisch @raeeceip @Precious000 I have assigned you the issue ticket. Happy collaborating!

**Comment by gridhead** [comment]
@Freedisch, @raeeceip and @Precious000 - Any updates on this one?

**Comment by Freedisch** [comment]
> @Freedisch, @raeeceip and @Precious000 - Any updates on this one?

My bad, I actually forgot about this task, I'll take care of it ASAP

**Comment by Freedisch** [comment]
@raeeceip , @Precious000 Let me know if u have started working on it, we can collaborate and complete it ASAP, it's an easy fix

**Comment by gridhead** [comment]
@Freedisch please feel free to get started with your own implementation.

**Comment by Freedisch** [comment]
@gridhead, just to let u know that the current link for downloading meetbot logs archive  is not working [link](https://mega.nz/file/cJYykbKA#jJozcnIG-WzwlYVQUXF25lqM5A8PNl2knQObQrSpOSk)

**Comment by gridhead** [comment]
@Freedisch, yep I am aware. @sopyb recently revised the documentation https://github.com/fedora-infra/mote/blob/main/README.md.

**Comment by yummyweb** [comment]
Is this issue still active? A PR has been made already.
